react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 
-import Square from '../Square';
+import SquareList from '../SquareList';
+import { boardRows } from '../../constants';
 
-import styles from './styles.module.scss';
+function Board({ squares, onClick }) {
+  const [firstRow, secondRow, thirdRow] = boardRows;
 
-class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
-
-  render() {
-    const status = 'Next player: X';
-    return (
-      <div>
-        <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(3)}
-          {this.renderSquare(4)}
-          {this.renderSquare(5)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(6)}
-          {this.renderSquare(7)}
-          {this.renderSquare(8)}
-        </div>
-      </div>
-    );
-  }
+  return (
+    <fragment>
```

Comment:
What is this fragment? There's no HTML element called fragment that I'm aware of. If you wanted to use React's fragments, it's Fragment. Also, since some version of React, there's a shorthand to use them:
```
<>
</>
```